clang/test/AST/Interp/arrays.cpp

```diff
@@ -27,6 +27,9 @@ static_assert(foo[2][3] == &m, "");
 static_assert(foo[2][4] == nullptr, "");
 
 
+const int SomeInt[] = {1};
+int getSomeInt() { return *SomeInt; }
```

Comment:
```suggestion
constexpr int SomeInt[] = {1};
constexpr int getSomeInt() { return *SomeInt; }

static_assert(getSomeInt() == 1);
```
so we're testing the decayed pointer.